packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -100,9 +100,8 @@ abstract class FlutterCommand extends Command {
   // This is a field so that you can modify the value for testing.
   Validator projectRootValidator = () {
     if (!FileSystemEntity.isFileSync('pubspec.yaml')) {
-      printError('Error: No pubspec.yaml file found.\n'
-        'This command should be run from the root of your Flutter project.\n'
-        'Do not run this command from the root of your git clone of Flutter.');
+      printError('Error: No pubspec.yaml file found. This command should be run '
```

Comment:
It showed up for me when I wasn't in a project or the flutter repo. I'll add this back in.